### PR TITLE
Limiting fraud dataset to speed up doc builds

### PR DESCRIPTION
**Pull request by bchen1116**
Testing to see whether or not limiting the fraud dataset will cause the doc builds to pass

Notebook builds passed [here](https://readthedocs.com/projects/feature-labs-inc-evalml/builds/523373/) and [here](https://readthedocs.com/projects/feature-labs-inc-evalml/builds/523322/), and doc example is [here](https://evalml.alteryx.com/en/bc_fix_notebooks/user_guide/automl.html)

### CODE REVIEW

**Review by freddyaboulton — APPROVED** [review]
Thanks @bchen1116 !!

File: docs/source/user_guide/automl.ipynb [hunk 1]
```diff
@@ -58,7 +58,7 @@
    "source": [
     "import evalml\n",
     "from evalml.utils import infer_feature_types\n",
-    "X, y = evalml.demos.load_fraud(return_pandas=True)\n",
+    "X, y = evalml.demos.load_fraud(n_rows=1000, return_pandas=True)\n",
```
Comment (dsherry) [inline docs/source/user_guide/automl.ipynb:61]: 👍

**Review by dsherry — APPROVED** [review]
Thanks! 🚢

**Review by angela97lin — APPROVED** [review]
LGTM! :shipit:

**Pull request merged**